Dockerfile.build

```diff
@@ -0,0 +1,29 @@
+ARG VERSION=0.0.0
+
+# centos7 amd64 dependencies
+FROM --platform=linux/amd64 nvidia/cuda:11.8.0-devel-centos7 AS base-amd64
```

Comment:
Probably want to use a non-deprecated distro

Reply:
I'll switch these to use ubuntu. For the binaries we build on an oldest distros possible for compatibility